### PR TITLE
Add Laura Poitras to the names generator.

### PR DESCRIPTION
**Pull request by soulshake**

### CODE REVIEW

**Comment by jfrazelle** [comment]
Awesome! LGTM

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -245,6 +245,9 @@ var (
 		// Henri Poincaré made fundamental contributions in several fields of mathematics. https://en.wikipedia.org/wiki/Henri_Poincar%C3%A9
 		"poincare",
 
+		// Laura Poitras is a director and producer whose work, made possible by open source crypto tools, advances the causes of truth and freedom of information by reporting disclosures by whistleblowers such as Edward Snowden. http://en.wikipedia.org/wiki/Laura_Poitras
```
Comment (tianon) [inline pkg/namesgenerator/names-generator.go]: Can you switch the URL to https please? :+1:

**Comment by tianon** [comment]
+1 :smile:

**Comment by tianon** [comment]
LGTM!

**Pull request merged**